### PR TITLE
Day10 puzzle part1

### PR DESCRIPTION
**Pull request by peter-sushko**
Hi Tim,

Here is the solution for problem 10 part 1. Let me know what you think. I'm curious how you solved it!

### CODE REVIEW

File: day10/part_1.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import numpy as np
+
+def find_start():
+    """Find the location of the 'S' symbol."""
+    with open('data.txt', 'r', encoding='utf-8') as sample_file:
+        for sample_index, sample_row in enumerate(sample_file):
+            if sample_row.find('S') != -1:
+                return sample_index + 1, sample_row.sample_index('S')
+
+pipe_map = np.empty((140, 140), dtype=str) # This 2d numpy array will act as a map.
```
Comment (tshur) [inline day10/part_1.py:14]: I would pull the 140 into a constant: either kInputWidth or INPUT_WIDTH or some other naming. Then, it also acts as documentation that 140 is a hardcoded dimension related to your input file.

File: day10/part_1.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        return new_x, new_y
+
+    if pipe_map[y_coord][x_coord] in ['J', 'F']:
+        new_x, new_y = assymmetrical_turn(x_step, y_step)
+        return new_x, new_y
+
+COUNTER = 1
+# Starting at the coordinate 41, 110 which is right above the 'S'.
+y = 40
+x = 111
```
Comment (tshur) [inline day10/part_1.py:86]: Aw, you didn't use your `find_start(...)` function.

File: day9/data.txt
Comment (tshur) [inline day9/data.txt:1]: beware getting other code changes mixed up into the wrong PRs! It will also make merging harder :p

**Review by tshur — APPROVED** [review]

File: day10/part_1.py (hunk 1 above)
Comment (peter-sushko) [reply day10/part_1.py:14]: I agree, I started doing it recently

**Pull request merged**